src/interfaces/index.ts

```diff
@@ -164,6 +165,7 @@ type SSL = SslSecurity | SslPfxSecurity | SSLReject;
 export interface SoapBaseClientConfig extends MCBaseClientConfig {
   type: 'soap';
   wsdl?: string;
+  overwriteEndpoint?: string;
```

Comment:
Please add two tests that use this config. One as ClientGenerator.get and the other one as a new soap monoclient

Reply:
@rcabralyu ready!